src/wovnio/wovnphp/custom_domain/CustomDomainLangs.php

```diff
@@ -80,7 +82,8 @@ public function computeSourceVirtualUrl($physicalUri, $lang, $defaultLang)
     {
         $currentLangDomainLang = $this->getSourceCustomDomainByLang($lang);
         $defaultCustomDomainLang = $this->getCustomDomainLangByLang($defaultLang);
-        return CustomDomainLangUrlHandler::changeToNewCustomDomainLang($physicalUri, $currentLangDomainLang, $defaultCustomDomainLang);
+        $virtualUrl = CustomDomainLangUrlHandler::changeToNewCustomDomainLang($physicalUri, $currentLangDomainLang, $defaultCustomDomainLang);
+        return $this->removePort($virtualUrl);
```

Comment:
What's the significance of removing the port here? According to the issue it says the problem is html swapper is not receiving the correct URL with the port number, which we are removing here.

Reply:
We want to send the page to html-swapper as a portless one to avoid having duplicated pages on WOVN's dashboard. This PR is just a temporary fix to make WOVN.php's side of things working, mostly. To fully make things work it would involve changes to foundation.

Reply:
I don't think we should be making a temporary fix here then. It is much harder to make changes to wovn.php than equalizer.

Reply:
This wouldn't conflict with our ultimate goal, which is to make libraries port sensitive but not equalizer, right? It's sorta like having frontend does a validation while backend also does its own validation.

Reply:
Equalizer should be the one deciding if port numbers are relevant or not when it comes to whether URLs are equivalent. The library should be providing as much information as possible to make that decision.

Reply:
Ok. Will remove this behavior.